src/fund/jsonTypes.ts

```diff
@@ -0,0 +1,25 @@
+/**
+Matches a JSON object.
+This type can be useful to enforce some input to be JSON-compatible or as a super-type to be extended from. Don't use this as a direct return type as the user would have to double-cast it: `jsonObject as unknown as CustomResponse`. Instead, you could extend your CustomResponse type from it to ensure your type only uses JSON-compatible types: `interface CustomResponse extends JsonObject { … }`.
+@category JSON
+*/
+export type JsonObject = { [Key in string]?: JsonValue };
```

Comment:
I thought these were a specific set up types and added them separately the same way as it was in CBPay-js so that types.ts does not get bloated but I can move these if that is not a concern.

Reply:
Yeah types bloating doesn't seem like a big concern, the swap component has a 300+ line types file. I would typically advocate for smaller files in other repos, but I think it's better to be consistent in this repo.